src/Api/Domain.php

```diff
@@ -125,6 +126,12 @@ public function create(string $domain, string $smtpPass = null, string $spamActi
             $params['ips'] = join(',', $ips);
         }
 
+        if (!empty($webScheme)) {
+            Assert::stringNotEmpty($webScheme);
```

Comment:
So, according to the API Documentation page, we have this:

![image](https://user-images.githubusercontent.com/7809429/112485785-32ebc600-8d73-11eb-9568-e6b340b31592.png)

At the moment, the `$webScheme` is quite open, as it allows "any" string. Ideally, it should be aligned to the two potential options, and we can set the by-default value too as part of the method's argument definition.

Please note the automated tests are not covering this update. They should be provided.

Cheers,